index.js

```diff
@@ -229,10 +228,27 @@ module.exports = class BemCell {
     }
 
     /**
+     * Determines whether specified cell is deep equal to cell or not
+     *
+     * @param  {BemCell} cell - the cell to compare
+     * @returns {Boolean} - A boolean value indicating whether or not specified cell is deep equal
```

Comment:
What a about remove line after describing what it returns ? cause it's seem obvious